thriftpy2/http.py

```diff
@@ -178,7 +194,10 @@ def __init__(self, uri, timeout=None, ssl_context_factory=None):
             self.path += '?%s' % parsed.query
         self.__wbuf = BytesIO()
         self.__http = None
-        self.__custom_headers = None
+        if http_header_factory:
```

Comment:
`self._http_header_factory = http_header_factory or THttpHeaderFactory()` is more readable here.

Reply:
done